lib/quickbooks/service/base_service.rb

```diff
@@ -289,7 +289,7 @@ def check_response(response, options = {})
         when 429
           message = parse_intuit_error[:message]
           raise Quickbooks::TooManyRequests, message
-        when 502, 503, 504
```

Comment:
Why the removal of the 502 code here?

Reply:
These updates came from the 0.6.2 version of the gem. I merged them in from the gem as I was surprised that they were not in the github repository. Are you the only owner of the gem, as it seems to have been updated independently of the github repository?

Reply:
Ah gotcha. Yep, my bad. After cutting `0.6.2` back in May I forgot to push my change to `version.rb`; which I just did. However, it was only `version.rb` and `HISTORY.md` that didnt get pushed. All other files (such as `lib/quickbooks/service/base_service.rb` and the addition of the `502` status check have been in the repo since May)

Reply:
These have now been reverted and can be ignored. I was confused by the status of the gem as it didn't match the repo. Do I need to do anything more for this pull request?